### PR TITLE
{bio}[foss/2019b] STAR v2.5.3a

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/ea2be6391ac84f325528b4dbf48d4621 for a full test report.

File: easybuild/easyconfigs/s/STAR/STAR-2.5.3a-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+easyblock = 'MakeCp'
+
+name = 'STAR'
+version = '2.5.3a'
+
+homepage = 'https://github.com/alexdobin/STAR'
+description = "STAR aligns RNA-seq reads to a reference genome using uncompressed suffix arrays."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```
Comment (Micket) [inline easybuild/easyconfigs/s/STAR/STAR-2.5.3a-foss-2019b.eb:17]: I don't think there is a MPI or BLAS dep here, so it should be dropped to GCC level.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node356.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/9c76cab6669912e0bf795d5e0572bea3 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**